docs/reporting/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Reporting
+nav_order: 34
+has_children: true
+has_toc: false
+---
+
+# Reporting
+
+The Kibana **Reporting** feature allows you to generate, schedule and share reports on-demand or based on an automated schedule. On-demand reports can be instantly downloaded in PNG, PDF and CSV formats, while schedule-based reports can be configured and set up for automated periodic delivery to stakeholders. You can do this either from the **Reporting** tab in left navigation panel or from within the **Dashboard**, **Visualization** and **Discover** tabs. To download, schedule and share reports, you must have the required role permissions. 
+
+## Create Report
+
+### Create a Report from the Dashboard, Visualize or Discover Page
+
+To create a new report from the Dashboard, Visualize or Discover page, do the following:
+
+1. From the top navigation bar, choose **Reporting**. The reporting dropdown menu allows you to generate an on-demand report and download it instantly as a PDF, PNG, or CSV file. 
+2. Choose **Download PDF** or **Download PNG** to generate on-demand reports in these formats from the Dashobard or Visualize pages. For a saved search report from the Discover page, choose **Download CSV**, since this is the default file format. You can close the report generating pop-up, while the download continues asynchronously in the background. A notification pop-up confirms that your report is generated successfully. This may take a few minutes based on the size of your dashboard, visualization or saved search.
+3. To create a schedule based report from any of the above pages, choose **Create report definition**. You will be navigated to the **Create report definition** page, where the report source is preselected and the time range is what was entered within the Dashboard, Visualize or Discover page. To continue, follow the instructions from step 6 in the next section.
+
+### Create a Report using the Report Definition
+
+Report definition allows you to define various settings, triggers, and delivery options for your report. To create a report definition, do the following:
```

Comment:
Let's make it clear that this section is for automated reports, periodically delivered to your inbox. No one cares about settings and triggers, etc. They care about using this process to get an email once a week.

Reply:
done